estate/views/estate_properties_view.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="estate_estate_action" model="ir.actions.act_window">
+        <field name="name">Estate</field>
+        <field name="res_model">estate.estate</field>
+        <field name="view_mode">list,form</field>
+        <field name="search_view_id" ref="estate.estate_estate_search"/>
```

Comment:
You're just going to have an issue with this line having a ref is that since the record you're refering to doesn't exist yet when this record is created, you're gonna have a parsing error. The `estate_estate_search` view should be defined before this one :+1: